### PR TITLE
add pnpm support

### PR DESCRIPTION
**Pull request by deini**
Closes https://github.com/parshap/check-node-version/issues/50

### CODE REVIEW

File: tools.js [hunk 1]
```diff
@@ -29,6 +29,12 @@ module.exports = {
      return `To install yarn, see https://github.com/yarnpkg/yarn/releases/tag/v${v}`;
     }
   },
+  pnpm: {
+    getVersion: "pnpm --version",
+    getInstallInstructions(v) {
+     return `To install npm, run \`npm install -g pnpm@${v}\``;
```
Comment (icatalina) [inline tools.js:35]:
```suggestion
     return `To install pnpm, run \`npm install -g pnpm@${v}\``;
```

**Review by parshap — APPROVED** [review]
This looks great! Thank you!

**Comment by parshap** [comment]
Published in v4.2.0.

File: tools.js (hunk 1 above)
Comment (deini) [reply tools.js:35]: Hehe my bad

Comment (parshap) [reply tools.js:35]: Ah, missed this. I will push a fix as v4.2.1. Thank you.

**Pull request merged**